resources/application-connector/values.yaml

```diff
@@ -87,7 +87,7 @@ application_connectivity_certs_sync:
       namespace: istio-system
   image:
     repository: eu.gcr.io/kyma-project/tpi/k8s-tools
-    tag: "20210407-86c769bd"
```

Comment:
There is a bonus space. Will it be a problem?

Reply:
Thank you, fixed